From 76693d5d214fab5e6a602956a73ea88f45598eac Mon Sep 17 00:00:00 2001
From: Alok G Singh <alephnull@hcoop.net>
Date: Tue, 19 Mar 2024 10:53:55 +0800
Subject: [PATCH] Tag @devops only for templated code

---
 .github/CODEOWNERS     | 3 ---
 .github/dependabot.yml | 2 --
 2 files changed, 5 deletions(-)

diff --git a/.github/CODEOWNERS b/.github/CODEOWNERS
index 8bf7fde..b95dd6a 100644
--- a/.github/CODEOWNERS
+++ b/.github/CODEOWNERS
@@ -1,5 +1,2 @@
 /ci/ @TykTechnologies/devops
 .github/workflows/release.yml @TykTechnologies/devops
-.github/workflows/sync-automation.yml @TykTechnologies/devops
-.github/workflows/pac.yml @TykTechnologies/devops
-/repo-policy/ @TykTechnologies/devops
\ No newline at end of file
diff --git a/.github/dependabot.yml b/.github/dependabot.yml
index ef85342..b756d59 100644
--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -9,8 +9,6 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-    reviewers:
-      - "TykTechnologies/devops"
 
   - package-ecosystem: "gomod"
     # Look for `go.mod` file in the `root` directory
-- 
2.40.1

